File: alzir/Entregável #1 - Algoritmos/JavaScript/primo.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Digite um número n: ", (num) => {
  if (num <= 0) {
    console.log("Digite um número inteiro positivo!");
  } else {
    let divisores = 0;

    for (let cont = 1; cont <= num; cont++) if (num % cont == 0) divisores++;

    if (divisores === 2) console.log(`${Number(num)} é número primo!`);
    else console.log(`${Number(num)} não é número primo!`);
  }
  readline.close();
});

File: alzir/Entregável #1 - Algoritmos/Java/src/Primo.java
import java.util.Scanner;

public class Primo {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Digite um número n: ");
    int num = sc.nextInt();
    int divisores = 0;

    for (int cont = 1; cont <= num; cont++) {
      if (num % cont == 0) {
        divisores++;
      }
    }

    if (divisores == 2) {
      System.out.println(num + " é número primo!");
    } else {
      System.out.println(num + " não é número primo!");
    }
    sc.close();
  }
}
